tests/test_modeling_bart.py

```diff
@@ -1,5 +1,5 @@
 # coding=utf-8
-# Copyright 2020 The HuggingFace Team. All rights reserved.
+# Copyright Fairseq Authors and The HuggingFace Inc. team. All rights reserved.
```

Comment:
No year? Plus, did the Fairseq authors really write the tests? I'm good with them being on the modeling files but tests are really just HF, no?